tests/unit/exporters/test_console_error_exporter.py

```
@@ -0,0 +1,117 @@
+        completed=0,
+        start_ns=0,
+        end_ns=1,
+        successful_request_count=0,
+        error_request_count=sum(e.count for e in error_summary or []),
+        error_summary=error_summary or [],
+    )
+    defaults.update(kwargs)
+    return ProfileResults(**defaults)
+
+
+def make_error(code=404, type="Not Found", message="Not Found", count=1):
```

Ruff (0.16.2)

[error] Function argument `type` is shadowing a Python builtin

(A002)